eclair-core/src/test/scala/fr/acinq/eclair/integration/IntegrationSpec.scala

```diff
@@ -296,32 +297,34 @@ class IntegrationSpec extends TestKitBaseClass with BitcoindService with AnyFunS
     // get the channelId
     sender.send(fundee.register, Symbol("channels"))
     val Some((_, fundeeChannel)) = sender.expectMsgType[Map[ByteVector32, ActorRef]].find(_._1 == tempChannelId)
+
     sender.send(fundeeChannel, CMD_GETSTATEDATA)
-    val channelId = sender.expectMsgType[HasCommitments].channelId
+    val channelId = sender.expectMsgType[RES_GETSTATEDATA[HasCommitments]].data.channelId
     awaitCond({
       funder.register ! Register.Forward(sender.ref, channelId, CMD_GETSTATE)
-      sender.expectMsgType[State] == WAIT_FOR_FUNDING_LOCKED
+      sender.expectMsgType[RES_GETSTATE[_]].state == WAIT_FOR_FUNDING_LOCKED
     })
 
     generateBlocks(bitcoincli, 6)
 
     // after 8 blocks the fundee is still waiting for more confirmations
-    fundee.register ! Register.Forward(sender.ref, channelId, CMD_GETSTATE)
-    assert(sender.expectMsgType[State] == WAIT_FOR_FUNDING_CONFIRMED)
+    //fundee.register ! Register.Forward(sender.ref, channelId, CMD_GETSTATE)
+    new EclairImpl(fundee).channelInfo(Left(channelId))(20 seconds).pipeTo(sender.ref)
```

Comment:
nit: not sure this change is really useful to do in this PR, maybe later? Even though I'm more in favor of improving the tests of `EclairImpl` independently of the `IntegrationSpec`.

Reply:
Done with f24cc1002.